### PR TITLE
nia (short) hairstyle + new ponytail hairstyle

### PR DESCRIPTION
**Pull request by SonixApache**
It's international homo awareness month so as such here I am with some new unisex hairstyles;
Nia (Short) and Medium Ponytail

female, Nia (Short) <-> male, Nia (Short)
femnia_malenia_femMedium pony_maleMedium pony
female, Medium Ponytail <-> male, Medium Ponytail

Nia (Short) is around as poofy around the top as the original, but cut down by a lot.
niahelmetcomparison

The Medium Ponytail is essentially just a shorter version of the Long Ponytail, except with some shading that I took off the Nia hair pallette.

othercolors

🆑 
 - imageadd: Added two new hairstyles; Nia (Short) and Medium Ponytail

### CODE REVIEW

**Comment by juanototo2** [comment]
Waifu buff.

**Comment by SonixApache** [comment]
OP updated with some tweaks

**Comment by PavelCIA** [comment]
fetish content

**Comment by ghost** [comment]
humans are fetish content since most people want to fuck them, remove humans

File: code/modules/mob/new_player/sprite_accessories.dm [hunk 1]
```diff
@@ -128,6 +128,10 @@
 /datum/sprite_accessory/hair/longpony
 	name = "Long Ponytail"
 	icon_state = "hair_longpony"
+	
+/datum/sprite_accessory/hair/gaypony
+	name = "Gay Ponytail"
```
[the remark this answers is not in the document]
Comment (SonixApache) [reply code/modules/mob/new_player/sprite_accessories.dm]: give me a better name and i'll change it

**Comment by zakanater19** [comment]
"It's international homo awareness month so as such"

fucking faggots

**Comment by SonixApache** [comment]
it remains a funny coincidence

**Pull request merged**